apps/condo/domains/notification/utils/serverSchema/index.js

```diff
@@ -46,9 +52,26 @@ async function resendMessage (context, data) {
     })
 }
 
+async function registerPushToken (context, data) {
+    if (!context) throw new Error('no context')
+    if (!data) throw new Error('no data')
+    if (!data.sender) throw new Error('no data.sender')
+    // TODO(codegen): write registerPushToken serverSchema guards
```

Comment:
Remove this TODO, if you've fulfilled it's requirement.

Reply:
fixed